discourse_theme.gemspec

```diff
@@ -5,39 +5,39 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "discourse_theme/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "discourse_theme"
-  spec.version       = DiscourseTheme::VERSION
-  spec.authors       = ["Sam Saffron"]
-  spec.email         = ["sam.saffron@gmail.com"]
-
-  spec.summary       = %q{CLI helper for creating Discourse themes}
-  spec.description   = %q{CLI helper for creating Discourse themes}
-  spec.homepage      = "https://github.com/discourse/discourse_theme"
-  spec.license       = "MIT"
-
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
-  end
-
-  spec.bindir        = "bin"
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.name = "discourse_theme"
+  spec.version = DiscourseTheme::VERSION
+  spec.authors = ["Sam Saffron"]
+  spec.email = ["sam.saffron@gmail.com"]
+
+  spec.summary = "CLI helper for creating Discourse themes"
+  spec.description = "CLI helper for creating Discourse themes"
+  spec.homepage = "https://github.com/discourse/discourse_theme"
+  spec.license = "MIT"
+
+  spec.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+
+  spec.bindir = "bin"
+  spec.executables = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.5.0'
+  spec.required_ruby_version = ">= 3.0.0"
 
   spec.add_runtime_dependency "minitar", "~> 0.6"
   spec.add_runtime_dependency "listen", "~> 3.1"
   spec.add_runtime_dependency "multipart-post", "~> 2.0"
   spec.add_runtime_dependency "tty-prompt", "~> 0.18"
   spec.add_runtime_dependency "rubyzip", "~> 1.2"
 
-  spec.add_development_dependency "bundler", "~> 2.0"
-  spec.add_development_dependency "rake", "~> 13.0"
-  spec.add_development_dependency "minitest", "~> 5.0"
-  spec.add_development_dependency "guard", "~> 2.14"
-  spec.add_development_dependency "guard-minitest", "~> 2.4"
-  spec.add_development_dependency "webmock", "~> 3.17"
```

Comment:
Not sure why we pin these for, I removed it because there isn't a good reason to do so IMO.